test/cypress/integration/consumer-tools/before-you-claim/before-you-claim.cy.js

```diff
@@ -27,4 +21,1007 @@ describe('Planning your Social Security', () => {
     cy.wait('@getSpanish');
     cy.url().should('contain', '/es');
   });
+
+  it('should display tool tip in step one after clicking svg tip icon', () => {
+    cy.get('#tooltip-container').should('not.be.visible');
+    cy.get(
+      '#step-one-form .content-l .content-l_col .input-labels span.input-labels_block span svg'
```

Comment:
What do you think about shortening the selector here to `#step-one-form .cf-icon-svg__help-round`? 

If we can avoid structural CSS (`.content-l .content-l_col`) in the selector call, that would be preferable.